paywall/src/stories/interface/Layout.stories.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { Provider } from 'react-redux'
```

Comment:
Are there components in `Layout` other than the header that rely on the state? The test only passes in the router state, which isn't used in the header, so we can probably get rid of it here.

Reply:
yep, just removed it, good catch, PTAL